src/Renderer/LayeredMaterial.js

```diff
@@ -146,11 +160,56 @@ const LayeredMaterial = function LayeredMaterial(id) {
     // Light position
     this.uniforms.lightPosition = new THREE.Uniform(new THREE.Vector3(-0.5, 0.0, 1.0));
 
-    this.setUuid(id || 0);
-    this.wireframe = false;
-    // this.wireframe = true;
-
     this.colorLayersId = [];
+    this.elevationLayersId = [];
+
+    if (material instanceof LayeredMaterial) {
+        // Getting inheritance from elevation
+        if (material.getElevationLayerLevel() > -1) {
+            this.textures[l_ELEVATION][0] = material.textures[l_ELEVATION][0];
+            this.offsetScale[l_ELEVATION][0] = offsetToParent(coordDestination.WGS84G[1], this.textures[l_ELEVATION][0].coordWMTS.zoom);
+            this.loadedTexturesCount[l_ELEVATION] = 1;
+            this.elevationLayersId = material.elevationLayersId;
+        }
+
+        // Getting inheritance from lighting
+        this.uniforms.lightPosition.value = material.uniforms.lightPosition.value;
+        this.uniforms.lightingEnabled.value = material.uniforms.lightingEnabled.value;
+
+        // Getting inheritance from color
+        let offsetTextures = 0;
+        for (let i = 0; i < material.colorLayersId.length; i++) {
+            const layerId = material.colorLayersId[i];
+            if (material.getColorLayerLevelById(layerId) > -1) {
```

Comment:
The whole loop could benefit from a comment explaining its purpose (and "how" it is achieved).